editor/src/core/workers/parser-printer/parser-printer.ts

```diff
@@ -1940,6 +1942,11 @@ export function lintAndParse(
   shouldTrimBounds: 'trim-bounds' | 'do-not-trim-bounds',
   applySteganography: SteganographyMode,
 ): ParsedTextFile {
+  const fileExtension = getFileExtension(filename)
+  if (!FileExtensionsToParse.includes(fileExtension.toLowerCase())) {
```

Comment:
Does `getFileExtension` include the `.`?
I'd say it's better to use `isJsFile` declared here since that also includes mjs and cjs files <https://github.com/concrete-utopia/utopia/blob/c2724ce54a8b6ca8db5f40c1672e8f4871966b89/editor/src/core/shared/file-utils.ts#L246-L253>

Reply:
thanks, pushed the change in https://github.com/concrete-utopia/utopia/pull/5247/commits/4399d71f4e134a9f482b660b1564d34564271940